plugins/defaultplugins/aclplugin/vppdump/dump_vppcalls.go

```diff
@@ -122,73 +281,160 @@ func DumpMacIPAcl(log logging.Logger, vppChannel *govppapi.Channel, timeLog meas
 		}
 	}()
 
+	aclMACIPRules := make(map[ACLIdentifier][]acl_api.MacipACLRule)
+	var wasErr error
+
 	req := &acl_api.MacipACLDump{}
 	req.ACLIndex = 0xffffffff
 	reqContext := vppChannel.SendMultiRequest(req)
 	for {
-		msg := &acl_api.MacipACLDump{}
+		msg := &acl_api.MacipACLDetails{}
 		stop, err := reqContext.ReceiveReply(msg)
 		if err != nil {
-			return err
+			log.Error(err)
+			wasErr = err
 		}
 		if stop {
 			break
 		}
-		log.Info(msg.ACLIndex)
+
+		identifier := ACLIdentifier{
+			ACLIndex: msg.ACLIndex,
+			Tag:      string(bytes.Trim(msg.Tag, "\x00")),
+		}
+
+		aclMACIPRules[identifier] = msg.R
+	}
+	return aclMACIPRules, wasErr
+}
+
+// DumpInterfaceAcls finds interface in VPP and returns its ACL configuration
+func DumpInterfaceAcls(log logging.Logger, swIndex uint32, vppChannel *govppapi.Channel,
+	timeLog measure.StopWatchEntry) (acl.AccessLists, uint8, error) {
+
+	log.Info("DumpInterfaceAcls")
+	// ACLInterfaceListDump time measurement
+	alAcls := acl.AccessLists{
+		Acl: []*acl.AccessLists_Acl{},
+	}
+
+	start := time.Now()
+	defer func() {
+		if timeLog != nil {
+			timeLog.LogTimeEntry(time.Since(start))
+		}
+	}()
+
+	res, err := vppcalls.DumpInterface(swIndex, vppChannel, timeLog)
+	log.Infof("Res: %+v\n", res)
+	if err != nil {
+		return alAcls, 0, err
+	}
+
+	if res.SwIfIndex != swIndex {
+		return alAcls, 0, fmt.Errorf(fmt.Sprintf("Returned interface index %d does not match request",
```

Comment:
The `fmt.Sprintf` seems redundant here, since `fmt.Errorf` already contains formatting.